packages/requests/src/lib/requests-cache.ts

```diff
@@ -70,10 +80,17 @@ export function isRequestCached<S extends StateOf<typeof withRequestsCache>>(
   key: string | number | string[] | number[],
   options?: { value?: CacheState['value'] }
 ) {
+  const now = Date.now();
+
   return function (state: S) {
     const type = options?.value ?? 'full';
     return coerceArray(key).some(
-      (k) => getRequestCache(k)(state).value === type
+      (k) => {
+        const cachedValue = getRequestCache(k)(state);
+        if (cachedValue.localStorageTimeStamp && cachedValue.localStorageTimeStamp < now) {
```

Comment:
How do we reset the cache? In the current situation, I will always get false when the time is up.